syntax = "proto3";

package api;

message SignUpRequest {
  string phone = 1;
  string username = 2;
  string password = 3;
  string email = 4;
}

message LoginRequest {
  string phone = 1;
  string username = 2;
  string password = 3;
  string email = 4;
}

message CheckTokenRequest {
  string token = 1;
}

message CommonUserResponse {
  string username = 1;
  string token = 2;
}
